services/intersection-api/Dockerfile

```diff
@@ -0,0 +1,116 @@
+# === BUILDER IMAGE for ACM ===
+# FROM alpine:3.12 as builder
```

Comment:
Suggestion (non-blocking): Consider removing the commented-out FROM lines in this file

Reply:
removed